README.md

```diff
@@ -1,2 +1,44 @@
-# wp-elements
-Predefined ACF groups
+# WP Elements
+
+> Predefined ACF groups like ContactDetails, SocialLinks or Hero. We will be adding more as we need them.
+
+
+## Getting Started
+
+The easiest way to install this package is by using composer from your terminal:
+
+```bash
+composer require moxie-lean/wp-elements
+```
+
+Or by adding the following lines on your `composer.json` file
+
+```json
+"require": {
+  "moxie-lean/wp-elements": "dev-master"
+}
+```
+
+This will download the files from the [packagist site](https://packagist.org/packages/moxie-lean/wp-elements)
+and set you up with the latest version located on master branch of the repository.
+
+After that you can include the `autoload.php` file in order to
+be able to autoload the class during the object creation.
+
+```php
+include '/vendor/autoload.php';
+```
+
+
+## Usage
+
+You need to register the elements you want to use using the ```Elements::init()``` function. This function takes the following parameters: $location and $element.
```

Comment:
Yes no need to change it, it was more like a tip. Yes the mastering markdown uses 3 but for code that is used between text like; 

Paragraph 1 

```
Code Here
```

Paragraph 2

Instead. And if you use the UI from GH you get just 1 when you use the code button.


> On Apr 4, 2016, at 3:14 PM, Caridad Zapatero Contreras <notifications@github.com> wrote:
> 
> In README.md <https://github.com/moxie-leean/wp-elements/pull/1#discussion_r58440170>:
> 
> > +```
> > +
> > +This will download the files from the [packagist site](https://packagist.org/packages/moxie-lean/wp-elements)
> > +and set you up with the latest version located on master branch of the repository.
> > +
> > +After that you can include the `autoload.php` file in order to
> > +be able to autoload the class during the object creation.
> > +
> > +```php
> > +include '/vendor/autoload.php';
> > +```
> > +
> > +
> > +## Usage
> > +
> > +You need to register the elements you want to use using the ```Elements::init()``` function. This function takes the following parameters: $location and $element.
> Let me change it, but I copied it from wp-widgets.
> Should we update it over there too?
> https://github.com/moxie-leean/wp-widgets/edit/master/README.md <https://github.com/moxie-leean/wp-widgets/edit/master/README.md>
> Also, if you follow the link about mardown formatting, you can see that 3 quotes are recommended (click on Code):
> https://guides.github.com/features/mastering-markdown/ <https://guides.github.com/features/mastering-markdown/>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub <https://github.com/moxie-leean/wp-elements/pull/1/files/b31a02190f3aa1e2f429df0b4049fc28f4887303#r58440170>